src/ZendViewRenderer.php

```diff
@@ -269,6 +269,11 @@ private function injectNamespacedResolver(AggregateResolver $aggregate) : void
         $aggregate->attach(new NamespacedPathStackResolver(), 0);
     }
 
+    /**
+     * @param AggregateResolver $aggregate
```

Comment:
Please undo all these changes as they are not needed. The method declaration already contains this information!
Additionally the topic of this PR the fix #68 rather than adding more DocBlocks or clean up anything.

That also makes the review easier.

Reply:
Sorry, I reverted the cleanup.